.github/workflows/vercel-deploy-erd-sample.yml

```diff
@@ -0,0 +1,92 @@
+# This workflow specifically deploys the `apps/erd-sample` app.
+# For deployments of other `apps/*` apps, see `.github/workflows/vercel-deploy.yml`.
+name: Vercel Deployment erd-sample
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup-deployment:
+    runs-on: ubuntu-latest
+    outputs:
+      environment: ${{ steps.environment.outputs.environment }}
+      has-changes: ${{ steps.changes.outputs.src }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: tj-actions/branch-names@v7.0.7
+        id: branch-name
+      - name: Detect Environment Changes
+        id: environment
+        run: |
+          echo "environment=${{ steps.branch-name.outputs.current_branch == 'main' && 'production' || 'preview' }}" >> $GITHUB_OUTPUT
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            src:
+              - './.github/workflows/vercel-deploy-erd-sample.yml'
+              - './frontend/apps/erd-sample/**'
+              - './frontend/packages/cli/**'
+              - './frontend/packages/db-structure/**'
+              - './frontend/packages/erd-core/**'
+              - './frontend/packages/ui/**'
+
+  deploy:
+    name: Deploy
+    needs: [setup-deployment]
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    if: ${{ needs.setup-deployment.outputs.has-changes == 'true' }}
+    strategy:
+      matrix:
+        apps:
+          - name: erd-sample
+            vercel-project-id-key: VERCEL_PROJECT_ID_ERD_SAMPLE
+    environment:
+      name: "${{ needs.setup-deployment.outputs.environment }} - ${{ matrix.apps.name }}"
+      url: ${{ steps.deployment.outputs.deployment-url }}
+    outputs:
+      app-name: ${{ matrix.apps.name }}
+      url: ${{ steps.deployment.outputs.deployment-url }}
+    env:
+      VERCEL_ORG_ID: ${{ vars.VERCEL_ORG_ID }}
+      VERCEL_PROJECT_ID: ${{ vars[matrix.apps.vercel-project-id-key] }}
+      working-directory: "${{ github.repository }}"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ env.working-directory }}
+      - uses: actions/checkout@v4
+        with:
+          path: "./mastodon/mastodon"
+          ref: "v4.3.2"
+          repository: "mastodon/mastodon"
+          sparse-checkout: "db/schema.rb"
+      - uses: ./liam-hq/liam/.github/actions/pnpm-setup
+        with:
+          working-directory: ${{ env.working-directory }}/frontend
+      - name: Install Vercel CLI
+        run: pnpm add --global vercel@latest
+      - name: Navigate frontend path
+        run: ln -s ${{ env.working-directory }}/frontend frontend
+      - name: Navigate mastodon path
+        run: ln -sf ../../../../../mastodon/mastodon/db/schema.rb ${{ env.working-directory }}/frontend/apps/erd-sample/schema.rb
```

Comment:
When the CLI `erd build` supports the `--input [URL]` option, this process will become simpler.